aten/src/ATen/SparseTensorUtils.h

```diff
@@ -129,4 +130,25 @@ inline LongTensor flatten_indices_by_dims(const LongTensor& indices, const IntAr
   return new_indices;
 }
 
+inline LongTensor coo_to_csr(const int64_t* indices, int64_t dim, int64_t nnz) {
```

Comment:
Add a comment briefly describing what this function does (the name is already good, but might as well)

Reply:
Seems like this should be defined next to csr_to_coo? Why is it in this header file?

Reply:
Yes it should be, however It was originally defined and used at `SparseTensorMath.cpp` to implement `sspaddmm`, so I moved to `SparseTensorUtils.h` to be also used to implement sparse-sparse matmul.